src/backendLogic.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const port = 5000;
+const fs = require('fs');
+const app = express();
+const path = require('path');
+const appRoot = path.resolve(__dirname);
+const cors = require('cors');
+
+app.use(express.json());
+app.use(cors());
+
+app.get('/', (req, res) => {
+    fs.readFile(`${path.parse(appRoot).dir}/data/workout_list.json`, 'utf-8', (err, data) => {
```

Comment:
let's change this with async module.
You can change it to something like below
```
const fileContent = await fs.readFileSync(`path`);
JSON.parse(fileContent);
....
```

Reply:
This should have look like this
```
fs.readFile(path, callback);
->
Promise(fs.readFile(path)).then(callback);
->
const fileContent = await fs.readFileSync(`path`);
```
Please note that you need to specify `async` to the function definition in order to use `await` keyword.

Reply:
fyi: https://blog.risingstack.com/mastering-async-await-in-nodejs/

Reply:
https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original

You can also change callback type of function into async-await usable function

Reply:
Changed